nexus-repository-p2/src/main/resources/static/rapture/NX/p2/app/PluginString.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+
+/*global Ext, NX*/
+
+/**
+ * P2 plugin strings.
+ */
+Ext.define('NX.p2.app.PluginStrings', {
+  '@aggregate_priority': 90,
+
+  singleton: true,
+  requires: [
+    'NX.I18n'
+  ],
+
+  keys: {
+    SearchP2_Group: 'R Repositories',
+    SearchP2_License_FieldLabel: 'License',
```

Comment:
fields above not used. Is it OK?

Reply:
Done 3ea35d536c7d0e7d6ca4309b09dc1e4a67d14d41